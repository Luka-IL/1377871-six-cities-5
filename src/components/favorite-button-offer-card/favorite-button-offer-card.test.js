import React from "react";
import renderer from "react-test-renderer";
import FavoriteButtonOfferCard from "./favorite-button-offer-card";

it(`Should FavoriteButtonOfferCard render correctly`, () => {
  const tree = renderer
    .create(
        <FavoriteButtonOfferCard
          onClickFavorite ={() => {}}
          favorite={true}
        >
          <React.Fragment />
        </FavoriteButtonOfferCard>
    ).toJSON();

  expect(tree).toMatchSnapshot();
});
