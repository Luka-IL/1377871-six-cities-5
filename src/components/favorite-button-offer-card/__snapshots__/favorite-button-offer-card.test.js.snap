// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should FavoriteButtonOfferCard render correctly 1`] = `
<button
  className="place-card__bookmark-button button place-card__bookmark-button--active"
  type="button"
>
  <svg
    className="place-card__bookmark-icon"
    height="19"
    onClick={[Function]}
    width="18"
  >
    <use
      xlinkHref="#icon-bookmark"
    />
  </svg>
  <span
    className="visually-hidden"
  >
    To bookmarks
  </span>
</button>
`;
